skyfield data has expired

Currently tests will fail because data used for test_skyfield_data_expiration has expired

```
FAILED tests/_test_msui/test_remotesensing.py::test_skyfield_data_expiration - AssertionError: [RuntimeWarning('The file finals2000A.all has expired. Please upgrade your version of `skyfield-data` or expect computation errors')]
assert 1 == 0
 +  where 1 = len(WarningsRecorder(record=True))
```

--- a/localbuild/meta.yaml
+++ b/localbuild/meta.yaml
@@ -42,7 +42,7 @@ requirements:
     - requests >=2.31.0
     - scipy
     - skyfield >=1.12
-    - skyfield-data >=5
+    - skyfield-data >=6
     - tk
     - owslib >=0.24
     - unicodecsv

